### ISSUE TITLE
regex parse error on windows

### ISSUE THREAD

**Issue by lagudomeze** [amethyst/tools#53]
~~~~~~~
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Syntax(

regex parse error:
]) (?<last>[^
     ^
error: unrecognized flag
)', libcore\result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
~~~~~~~

tools/src/templates.rs 119 line
~~~~~~~
Regex::new("(?<last>[^\r])\n")
~~~~~~~

##### REGEX example is
~~~~~~~
let re = Regex::new(r"(?P<last>[^,\s]+),\s+(?P<first>\S+)").unwrap();
let result = re.replace("Springsteen, Bruce", "$first $last");
assert_eq!(result, "Bruce Springsteen");
~~~~~~~

maybe **Regex::new("(?P<last>[^\r])\n")   ** is rignt?

**Comment by Xaeroxe** [comment]
Thanks for reporting this!  Working on a fix now.

**Comment by Xaeroxe** [comment]
Fix published in 0.7.1
